bundles/net.tourbook.ui.tests/.project

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>net.tourbook.ui.tests</name>
```

Comment:
I would prefer, that all test projects have the prefix `net.tourbook.tests` that all tests are grouped together by name, in this case `net.tourbook.ui.tests` -> `net.tourbook.tests.ui`

Reply:
By convention, and [from my experience](https://github.com/FJBDev/mytourbook/commit/99e889aea2cbe3d6f98ca04317f8f30cc392aee0), the bundle name needs to end with `.tests` in order to work. 

`(often a fragment of the host/target plugin with the suffix ".tests")`

[Source](https://wiki.eclipse.org/Tycho/Packaging_Types#eclipse-test-plugin)

Reply:
When `.tests` must be a postfix, then the current name is OK